src/components/pages/Home.vue

```diff
@@ -4,6 +4,26 @@
 			<img class="logo" src="static/klasa.svg">
 			<br>
 			<br>
+			<div class="unmaintainedBanner has-text-left">
+				<h2 class="title is-3">
+					<strong>
+						Klasa is not maintained anymore; here's why:
+					</strong>
+				</h2>
+				Due to the differences between us, the community, and one of the ex Core Developers,
+				Dirigeants and all its projects are now unable to be maintained, as most of the active
+				contributors and Core Developers were banned from the Discord server.
```

Comment:
```suggestion
				Dirigeants and all its projects are now unable to be maintained. Furthermore, most of the active contributors and Core Developers were banned from the Discord server.
```

Reply:
it's action and cause, not 2 different things